### DISCUSSION TITLE
[Docs] Inconsistencies in the `Input Variants` section of `Input` component docs

### DISCUSSION THREAD

**Discussion by ErykDarnowski** [GeekyAnts/NativeBase#5346] [General]
### Description

Described in the `Additional Information` section (as it was easier to format there)

### CodeSandbox/Snack link

https://docs.nativebase.io/input#h3-input-variants

### Steps to reproduce

1. Go to URL
2. See errors

### NativeBase Version

3.4.8

### Platform

- [ ] Android
- [ ] CRA
- [ ] Expo
- [ ] iOS
- [ ] Next

### Other Platform

No platform to specify

### Additional Information

These are the issues / inconsistencies:

1. The `Round` variant described in the docs isn't actually correct as the proper name is `rounded`
2. (this is actually a suggestion) I'm wondering why `Outline` isn't `Outlined`? This would deffenetiely be more consistent with the other variants!

PS.
It would be nice to have a `Documentation Issue` issue format available 😅

**Comment by ankit-tailor** [comment]
Hey @ErykDarnowski, Thanks for your suggestions. We'll look into it.

**Comment by ankit-tailor** [comment]
Hey @ErykDarnowski, Thanks for your suggestions, needs more discussion on this. Moving this issue to discussions.
